editor/plugins/script_editor_plugin.cpp

```diff
@@ -3031,8 +3053,19 @@ ScriptEditor::ScriptEditor(EditorNode *p_editor) {
 	script_split->add_child(list_split);
 	list_split->set_v_size_flags(SIZE_EXPAND_FILL);
 
+	scripts_vbox = memnew(VBoxContainer);
+	scripts_vbox->set_custom_minimum_size(Size2(0, 90));
```

Comment:
This seems pointless, as the min size of `script_list` should already do the job.

Reply:
removed that